### ISSUE TITLE
Accessibility Zoom doesn't work mac to mac

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#1063]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/1063
> 
> Issue created on: 2021-02-14 by @marcobazzani
> Issue last updated on: 2021-02-14

I have Ctrl+wheel used to zoom the screen,
on the guest system this command doesn't work with barrier mouse+keyboard while it works with guest mouse+kb

macos 11.2 both host and guest
barrier 2.3.3-release-3395cca9
